src/main/java/com/minecraftabnormals/upgrade_aquatic/core/mixin/DrownedModelMixin.java

```diff
@@ -0,0 +1,69 @@
+package com.minecraftabnormals.upgrade_aquatic.core.mixin;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import net.minecraft.client.renderer.entity.model.DrownedModel;
+import net.minecraft.client.renderer.entity.model.ZombieModel;
+import net.minecraft.entity.Pose;
+import net.minecraft.entity.monster.ZombieEntity;
+import net.minecraft.util.HandSide;
+import net.minecraft.util.math.MathHelper;
+
+@Mixin(DrownedModel.class)
+public class DrownedModelMixin<T extends ZombieEntity> extends ZombieModel<T> {
+
+	protected DrownedModelMixin(float p_i48914_1_, float p_i48914_2_, int p_i48914_3_, int p_i48914_4_) {
+		super(p_i48914_1_, p_i48914_2_, p_i48914_3_, p_i48914_4_);
+	}
+
+	@Inject(at = @At("TAIL"), method = "setRotationAngles(Lnet/minecraft/entity/monster/ZombieEntity;FFFFF)V")
+	public void setRotationAngles(T entityIn, float limbSwing, float limbSwingAmount, float ageInTicks, float netHeadYaw, float headPitch, CallbackInfo info) {
```

Comment:
In general, this method is fairly good. But there's still format errors at some lines, such as 32, 33, and 34. There are more, but I won't point out every line.

Another nitpick is that many of the local fields in this method are not named.